qa/sc_forger_feerate.py

```diff
@@ -0,0 +1,149 @@
+                am = amount - i*am
+            outputs.append({"publicKey": receiver, "value": am})
+
+        j = {"outputs": outputs, "fee": fee}
+
+        request = json.dumps(j)
+        ret = sc_node.transaction_sendCoinsToAddress(request)
+        pprint.pprint(ret)
+        try:
+            txid = ret["result"]["transactionId"]
+            return txid
+        except Exception as e:
+            print("Exception: " + e.error)
+
+
+    def run_test(self):
+        time.sleep(0.1)
+        self.sync_all()
+        mc_node = self.nodes[0]
+        sc_node = self.sc_nodes[0]
+
+        scblock_id = generate_next_blocks(sc_node, "first node", 1)[0]
+        print(scblock_id)
+
+        # create FT to SC for creating some utxo to spend
+        sc_address = sc_node.wallet_createPrivateKey25519()["result"]["proposition"]["publicKey"]
+        mc_return_address = mc_node.getnewaddress()
+
+        ft_args = [{
+            "toaddress": sc_address,
+            "amount": 10,
+            "scid": self.sc_nodes_bootstrap_info.sidechain_id,
+            "mcReturnAddress": mc_return_address
+        }, {
+            "toaddress": sc_address,
+            "amount": 20,
+            "scid": self.sc_nodes_bootstrap_info.sidechain_id,
+            "mcReturnAddress": mc_return_address
+        }, {
+            "toaddress": sc_address,
+            "amount": 30,
+            "scid": self.sc_nodes_bootstrap_info.sidechain_id,
+            "mcReturnAddress": mc_return_address
+        },
+         {
+            "toaddress": sc_address,
+            "amount": 40,
+            "scid": self.sc_nodes_bootstrap_info.sidechain_id,
+            "mcReturnAddress": mc_return_address
+        },
+         {
+            "toaddress": sc_address,
+            "amount": 50,
+            "scid": self.sc_nodes_bootstrap_info.sidechain_id,
+            "mcReturnAddress": mc_return_address
+        }]
+
+        mc_node.sc_send(ft_args)
+        assert_equal(1, mc_node.getmempoolinfo()["size"], "Forward Transfer expected to be added to mempool.")
+
+        # Generate MC block and SC block
+        mc_node.generate(1)
+        generate_next_blocks(sc_node, "first node", 1)
+        check_box_balance(sc_node, Account("", sc_address), "ZenBox", 5, 150)
+
+        sc_send_amount = 1  # Zen
+        print("\nSending {} satoshi ({} Zen) inside sidechain...".format(sc_send_amount * COIN, sc_send_amount))
+        sc_address = sc_node.wallet_allPublicKeys()["result"]["propositions"][-1]["publicKey"]
+
+        tx1 = self.send_coins(sc_node, sc_address, amount=111111, fee=100, numberOfOutputs=5)
+
+        tx2 = self.send_coins(sc_node, sc_address, amount=222222, fee=300, numberOfOutputs=5)
+
+        tx3 = self.send_coins(sc_node, sc_address, amount=333333, fee=100, numberOfOutputs=1)
+
+        tx4 = self.send_coins(sc_node, sc_address, amount=333333, fee=101, numberOfOutputs=1)
+
+        tx4 = self.send_coins(sc_node, sc_address, amount=333333, fee=99, numberOfOutputs=1)
```

Comment:
`tx4` declared twice, probably here it must be `tx5`

Reply:
Done